notebooks/README.md

```diff
@@ -13,7 +13,6 @@ documentation tree,
 ## Notebook Information
 Notebook Title | Data set(s) | Notebook Description | External Download (Size)
 --- | --- | --- | ---
-[NYC Taxi Years Correlation](nyc_taxi_years_correlation.ipynb) | [NYC Taxi Yellow 01/2016, 01/2017, taxi zone data](https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page) | Demonstrates using Point in Polygon to correlate the NYC Taxi datasets pre-2017 `lat/lon` locations with the post-2017 `LocationID` for cross format comparisons. | Yes (~3GB)
 [Stop Sign Counting By Zipcode Boundary](ZipCodes_Stops_PiP_cuSpatial.ipynb) | [Stop Sign Locations](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dstop) [Zipcode Boundaries](https://catalog.data.gov/dataset/tiger-line-shapefile-2019-2010-nation-u-s-2010-census-5-digit-zip-code-tabulation-area-zcta5-na) [USA States Boundaries](https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dadministrative) | Demonstrates Quadtree Point-in-Polygon to categorize stop signs by zipcode boundaries. | Yes (~1GB)
 [Taxi Dropoff Reverse Geocoding (GTC 2023)](Taxi_Dropoff_Reverse_Geocoding.ipynb) | [National Address Database](https://nationaladdressdata.s3.amazonaws.com/NAD_r12_TXT.zip) [NYC Taxi Zones](https://d37ci6vzurychx.cloudfront.net/misc/taxi_zones.zip) [taxi2015.csv](https://rapidsai-data.s3.us-east-2.amazonaws.com/viz-data/nyc_taxi.tar.gz) | Reverse Geocoding of 22GB of datasets in NYC delivered for GTC 2023 | Yes (~22GB)
```

Comment:
Does the Taxi Dropoff notebook still work?

Reply:
It's not currently being skipped in `ci/test_notebooks.sh` so I presume it should still be working

https://github.com/rapidsai/cuspatial/blob/f1bd2b3db62b4b1971b9cd07e9ccafbe1b595c40/ci/test_notebooks.sh#L32

Reply:
I see it's stored in our S3 bucket.